libpng causes issues in CI

All our CI tests fail with the same error:

    [ 32%] Performing download step (git clone) for 'libpng_ext'
    Cloning into 'libpng_ext'...
    fatal: unable to access 'https://git.code.sf.net/p/libpng/code/': gnutls_handshake() failed: Handshake failed
    Cloning into 'libpng_ext'...
    fatal: unable to access 'https://git.code.sf.net/p/libpng/code/': gnutls_handshake() failed: Handshake failed
    Cloning into 'libpng_ext'...
    fatal: unable to access 'https://git.code.sf.net/p/libpng/code/': gnutls_handshake() failed: Handshake failed
    -- Had to git clone more than once:
          3 times.
    CMake Error at /home/travis/build/UniStuttgart-VISUS/megamol-dev/build/core/libpng_ext-prefix/tmp/libpng_ext-gitclone.cmake:66 (message):
    Failed to clone repository: 'https://git.code.sf.net/p/libpng/code'

--- a/CMakeExternals.cmake
+++ b/CMakeExternals.cmake
@@ -102,8 +102,9 @@ function(require_external NAME)
     endif()
     ExternalProject_Get_Property(zlib_ext INSTALL_DIR)
     add_external_project(libpng_ext
-      GIT_REPOSITORY https://git.code.sf.net/p/libpng/code
+      GIT_REPOSITORY https://github.com/UniStuttgart-VISUS/libpng.git
       GIT_TAG "v1.6.34"
+      DEPENDS zlib_ext
       BUILD_BYPRODUCTS "<INSTALL_DIR>/${LIBPNG_DEBUG}" "<INSTALL_DIR>/${LIBPNG_RELEASE}"
       CMAKE_ARGS
         -DPNG_SHARED=OFF

